src/pages/Home/Home.css

```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-size: 20px;
+  text-decoration: none;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+.App {
+  background-color: #d24379;
+  min-height: 100vh;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.navbar h1 {
+  color: white;
+  font-family: 'Lato', sans-serif;
+  padding: 10px;
+  margin: 10px;
+}
+
+.searchbar {
+  display: flex;
+  gap: 10px;
+}
+
+.companyList {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin: 20px 0 0 0;
+}
+
+.companyListItem {
+  min-height: 150px;
+  background-color: #d24379;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-self: center;
+}
```

Comment:
- [ ] `optional`
- [ ] Amazing job you have done! 👏 
- [ ] However, you missed out some little things on the design, some buttons
- [ ] please see screenshot below 🥂  see the circled area 

`your template`
![image](https://github.com/Naledi-Dikgale/react-capstone/assets/66230306/14e8755b-08ce-477a-b575-2c82e223323f)

`original template`
![image](https://github.com/Naledi-Dikgale/react-capstone/assets/66230306/09a97038-2050-49fd-be94-eee0470da546)